### DISCUSSION TITLE
Running inference with a YOLO model using an edge TPU on Frigate

### DISCUSSION THREAD

**Discussion by Grego3j** [blakeblackshear/frigate#7243] [Ask A Question] [answered]
Hello All!

This is mostly a question to make sure that I have understood the documentation correctly. I have trained a YOLO model for a custom object detection task that I want to run on Frigate using an edge TPU. However, after reading the docs [here](https://docs.frigate.video/configuration/detectors), it seems like the base version of Frigate only supports YOLO running on a TensorRT detector. Is this understanding correct, or is there a simple way for a YOLO model to run on an edge TPU without modifications? 

Thanks for the help!

**Comment by NickM-27** [comment, marked as answer]
Yolo is supported for tensorrt and openvino. 

There are some PRs for adding yolo support to the coral but they're all very outdated with many conflicts.

**Reply by Grego3j** [reply]
Great! Thanks so much for the quick reply. In your opinion, if I want to use an edge TPU, would I be better off switching to something like a  SSD MobileNet model?

**Reply by NickM-27** [reply]
I think that would be the simplest approach

**Reply by trixor** [reply]
FWIW; I'm trying to see if I can get a tutorial together for a fine-tuned YoloV7 or YoloV8 Model on the TensorRT. I initially tried to get it to work on EdgeTPU Coral, but had to give up after a while. I probably spend more time on trying to make it work then I dare admit :(.  SSD MobileNet model do work on EdgeTPU fairly easily :).
